src/full-overlay-frame-helper.js

```diff
@@ -44,6 +47,7 @@ export function centerFrameUnderVsyncMutate(
     'width': px(iframeRect.width),
     'transition': `transform ${transitionTimeMs}ms ease`,
     'transform': translate(translateX, translateY),
+    'margin': 0,
```

Comment:
@lannka Added to keep margins from spilling into the centered view, causing another type of visual jump!